server.js

```diff
@@ -0,0 +1,30 @@
+const express = require('express');
+const package_info = require('./package.json');
+
+// create new express app and save it as "app"
+const app = express();
+
+// server configuration
+const PORT = 8080;
+const server_name = package_info.serverName;
```

Comment:
Can't we directly load it inside the  JSON creation?

Reply:
yes, done